src/live-search/live-search.md

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
This one also repeats in the URL. It's basically a mini landing page. 

maybe `live-search/sections` ?  We should come up with a standard for this kins of thing. Some of our existing URLs need work.

Reply:
i used the same format as SEO and Search
https://docs.magento.com/user-guide/marketing/seo-search.html

Reply:
Leave for now, but will have to figure out before we publish.